documentation/src/main/asciidoc/topics/config_examples/custom_interceptors.xml

```diff
@@ -0,0 +1,17 @@
+<local-cache name="cacheWithCustomInterceptors">
+  <!-- Define custom interceptors. -->
+  <!-- Custom interceptors must extend
```

Comment:
it is recommended but it isn't mandatory.
If you don't extend `BaseCustomAsyncInterceptor`, you will get a warning with message `Custom interceptor '%s' does not extend BaseCustomInterceptor, which is recommended`